homeassistant/bootstrap.py

```diff
@@ -408,6 +409,9 @@ def log_exception(ex, domain, config):
         message += " (See {}:{})".format(config.__config_file__,
                                          config.__line__ or '?')
 
+    message += '. Please check the docs at '
+    message += 'https://home-assistant.io/components/{}/ '.format(domain)
```

Comment:
Can you make it 1 string concatenation instead of 2 (slightly better performance). Python has implicit string concatenation within parentheses:

```python
if domain != 'homeassistant':
    message += ('. Please check the docs at '
                'https://home-assistant.io/components/{}/ '.format(domain))
```